ospfd/ospf_helper.c

```diff
@@ -0,0 +1,986 @@
+/*
```

Comment:
This is certainly not a good name for this file :)

How about ospf_gr.c or ospf_gr_helper.c ?

Reply:
Addressed.